models/CreateNewPatientRequest.js

```diff
@@ -0,0 +1,83 @@
+/*
+
+Patient {
+
+  family : string
+  given : string
+  suffix : string
+  phone : {
+    value : string
+    use : enum (home, work, temp, old, mobile)
+  }
+  email : array of strings
+  gender : enum (male, female, other, unknown)
+  birthdate : string
+  race : enum (American Indian or Alaskan Native, Asian, Native Hawaiian or Other Pacific Islander, Black or African-American, White, Other Race)
+  ethnicity: enum (Hispanic, Not Hispanic)
+  permissionToFollowup: boolean
+  address: {
+    line: string
```

Comment:
The UI has separate fields for Address Line 1 and Address Line 2.  Does this structure need to reflect that?  Or will we just be combining Address Line 1 and Address Line 2 into "line"?

Reply:
We will probably combine the two into one line since the FHIR resource doesnt take two address lines.